services/core-api/app/api/now_applications/models/administrative_amendments/application_reason_code.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy.dialects.postgresql import UUID
```

Comment:
Can we clean up the imports in this file? I don't think they're all being used.

Reply:
done